### DISCUSSION TITLE
Có roadmap ngắn nhất để học reactjs không ạ?

### DISCUSSION THREAD

**Discussion by pta201** [OngDev/community#173] [Q&A]
Có roadmap ngắn nhất để học reactjs không ạ?

**Comment by chuong113** [comment]
![react](https://user-images.githubusercontent.com/90631276/135017665-e3ce301c-7aab-4337-a399-9bf7f7d277a0.png)
Mình thầy roadmap này đầy đủ rồi. Bạn cứ từ từ mà học

**Comment by Minhbe-newbie** [comment]
Nếu bạn muốn nhanh có việc :Bạn nên bắt đầu xây dựng dự án dùng MUi, react và redux. Sau đó dùng nó để đi phỏng vấn thẳng cẳng luôn.. Còn nếu chỉ là học để biết thì sẽ tìm hiểu bên trên. Tôi dùng cách 1 sau này đi thực tập toàn phải fix hiển thị thôi, tốc độ thăng tiến chậm chạm nên out ra về tu luyện thằng typescript, clean code thuật toán Vim, .... Kiểu lùi 1 bước tiến 2 bước ấy
